UnitTests/GTMSessionFetcherFetchingTest.m

```diff
@@ -2443,6 +2585,49 @@ - (void)testStopFetchWithCallback_PreBeginStop_WithoutFetcherService {
   [self testStopFetchWithCallback_PreBeginStop];
 }
 
+- (void)testStopFetchNoCallback_DelayedStop {
+  // Wait after the fetch is started and then stop it.
+  __block XCTestExpectation *stoppingExpectation;
```

Comment:
do you need this block variable and the assignment?

Reply:
Probably not, removing.